### DISCUSSION TITLE
Random

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#36820] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey everyone,

I just want to say that I participated in HacktoberFest and all my PRs have been accepted.

I am thrilled to share the news that I have ordered a T-Shirt from HacktoberFest and a badge from Dev Community.

Don't wait for anyone guys. You still have time. All the best 👍🏻

**Comment by ettaboyle** [comment, marked as answer]
👋🏻  hi @iLakshya  Welcome to the GitHub Community! I wanted to make sure you were aware that the 'Discussions' category is actually for '[GitHub Discussions](https://github.com/features/discussions)' the product -- not general topics of conversation.

I'm going to move this post to the 'General' category for you but please keep this in mind for posts in the future. All the "product" categories are dedicated to GitHub Products.

Thank you! ✨
